src/ploomber/cli/task.py

```diff
@@ -63,6 +63,8 @@ def _task_cli(accept_task_id=False):
                 api.tasks_update(getattr(args, 'task_id'), 'failed')
             raise
         else:
+            print("{} executed successfully!".format(task.name))
```

Comment:
please change this to use `click.echo` it works better across operating systems, other than that, it looks good (just add some tests)

Reply:
Thanks for the feedback! I just changed it.